docs/orcfax-feeds/feed-overview.md

```diff
@@ -53,77 +35,144 @@ For a feed of type CER, the naming convention is
+feeds as business needs dictate. However, any change to service relating to
+subsidized feeds will be announced three months in advance in order to provide
+integrators with adequate forewarning.
+
+[showcase]: #showcase-feeds
+
+## Sponsored feeds
+
+When an integrator agrees to support a given feed through a heartbeat
+publication subscription, that feed will be designated as a sponsored feed for
+the duration of that agreement.
+
+While the integrator has committed to supporting the sponsored feed, and thereby
+facilitating its publication, projects which are not participating in the feed's
+sponsorship and yet utilize the on-chain data should do so at their own risk;
+feed sponsors may change or remove feed support at their own discretion and are
+not required to communicate these intentions to external parties.
+
+Orcfax has the following sponsored feeds:
+
+| Feed name | Start Date |
+| --------- | ---------- |
+| TBA       | TBA        |
+
+## Sunsetting sponsored feeds
+
+Projects which fail to adhere to subscription agreements risk disruptions to
+their feeds.
+
+Integrators will be given a two week window to come current on balances owing,
+afterwhich unpaid feeds will have publication suspended.
+
+## Showcase feeds
+
+In order to demonstrate the kinds of feeds which integrators can utilize in
+their dApps, Orcfax will periodically select feeds from its catalogue and
+highlight them through publication as showcase feeds. Orcfax sets no minimum or
+maximum number of showcase feeds that it may support.
+
+Orcfax is currently offering the following CEX showcase feeds:
+
+| Feed name | Start Date                 | End date |
+| --------- | -------------------------- | -------- |
+| ADA-BTC   | Sep 12th 2024 17:01:24 UTC | TBA      |
+| BTC-USD   | Sep 12th 2024 17:01:24 UTC | TBA      |
+
+Orcfax is currently offering the following DEX showcase feeds:
+
+| Feed name | Start Date                 | End date |
+| --------- | -------------------------- | -------- |
+| ADA-DJED  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| ADA-iUSD  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| ADA-USDM  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| AGIX-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| BOOK-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| BTN-ADA   | Sep 12th 2024 17:01:24 UTC | TBA      |
+| CERRA-ADA | Sep 12th 2024 17:01:24 UTC | TBA      |
+| COPI-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| HOSKY-ADA | Sep 12th 2024 17:01:24 UTC | TBA      |
+| HUNT-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| INDY-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| LENFI-ADA | Sep 12th 2024 17:01:24 UTC | TBA      |
+| LQ-ADA    | Sep 12th 2024 17:01:24 UTC | TBA      |
+| MIN-ADA   | Sep 12th 2024 17:01:24 UTC | TBA      |
+| SHEN-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| SNEK-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
+| WMTx-ADA  | Sep 12th 2024 17:01:24 UTC | TBA      |
```

Comment:
We still have this open https://github.com/orcfax/issues/issues/208 and I opened this https://github.com/orcfax/issues/issues/229.

In preparation for either, I would remove showcase feeds here entirely. 

The status feed is one route we have to announce a sunset -- todo -- check with Gabriel how we might run that protocol. 

We could consider some text such as:

> Feeds not currently listed as subsidized or sponsored are showcase. 

This ensures that:
* long-running feeds receive maintenance on these docs, and maintenance is a light-lift.
* short-running feeds are not a maintenance burden. 

I might not even explain that this will be visible on the explorer right now, relying instead on the strength of our communications until the attached issues are addressed.